cmd/aries-js-worker/src/aries.js

```diff
@@ -183,11 +181,16 @@ const Aries = function(opts) {
             createInvitation: async function (text) {
                 return invoke(aw, pending,  this.pkgname, "CreateInvitation", text, "timeout while creating invitation")
             },
-            receiveInvitation: async function (text) {
-                return invoke(aw, pending,  this.pkgname, "ReceiveInvitation", text, "timeout while receiving invitation")
+            receiveInvitation: async function (invitation) {
+                return invoke(aw, pending,  this.pkgname, "ReceiveInvitation", invitation, "timeout while receiving invitation")
             },
-            acceptInvitation: async function (text) {
-                return invoke(aw, pending,  this.pkgname, "AcceptInvitation", text, "timeout while accepting invitation")
+            acceptInvitation: async function (connectionID, publicDID="") {
```

Comment:
As per my opinion, we should avoid using strict function args, because every time there is an update in command controller models we may have to come back here to make corresponding updates.